<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotificationPopup</class>
 <widget class="QWidget" name="NotificationPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>180</width>
    <height>80</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="mainLayout" stretch="1,3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QWidget" name="headerWidget">
     <layout class="QHBoxLayout" name="headerLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string/>
        </property>
        <property name="fixedSize" stdset="0">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="styleSheet">
         <string>QPushButton { color: #888888; background: transparent; border: none; font-size: 16px; } QPushButton:hover { color: #d9534f; }</string>
        </property>
        <property name="text">
         <string>×</string>
        </property>
        <property name="fixedSize" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="contentWidget">
     <layout class="QVBoxLayout" name="contentLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="priorityLayout">
        <property name="spacing">
         <number>8</number>
        </property>
        <item>
        <widget class="QLabel" name="priorityLabel">
         <property name="fixedSize" stdset="0">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
        </widget>
        </item>
        <item>
        <widget class="QLabel" name="messageLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
          <string>color: #444444; font-size: 13px; font-weight: 600;</string>
          </property>
          <property name="text">
           <string>提醒内容</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
